ci-operator/step-registry/openshift-pipelines/install-and-tests/openshift-pipelines-install-and-tests-ref.yaml

```diff
@@ -11,5 +11,9 @@ ref:
       default: latest
       documentation: |-
         OLM channel selected for openshift-pipelines operator.
+    - name: ROSA
```

Comment:
I think this env var is eventually unused, maybe your first thought was to use it [in the login section](https://github.com/openshift/release/pull/47988/files#diff-7b19e927a3d5d53bfcef2d8421b10f3858470539ca43d2a476765a3fbf2719dfR23)?

Reply:
I noticed that, just wanted to leave it for future purpose, but you are right, I should just remove it for now